app/src/main/python/gogdl/dl/managers/dependencies.py

```
@@ -0,0 +1,166 @@
+        new_depots = []
+        if not self.ids:
+            return []
+        installed = set()
+
+        # This will be always None for redist writen in game dir
+        existing_manifest = None
+        if os.path.exists(self.installed_manifest):
+            try:
+                with open(self.installed_manifest, 'r') as f:
+                    existing_manifest = json.load(f)
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)